cmake/python/python_wrapper_helper.py.in

```diff
@@ -79,7 +79,7 @@ def generate_python_wrapper(header_directories, include_paths, library_name, cpp
         start_with_declarations=declarations)
 
     # Set include dirs and cflags to avoid warnings and errors
-    xml_generator_config.append_cflags("-std=c++11")
+    xml_generator_config.append_cflags("-std=c++@CMAKE_CXX_STANDARD@")
 
```

Comment:
Looks like older versions of compiler/OS have problems with some warnings on cast_xml calls; maybe that was the reason why we didn't update the flags originally.
I tested a bit around locally (ok, only have Ubuntu20.04 where the error isn't occurring), but maybe adding a warning suppression for the parser helps
xml_generator_config.append_cflags("-w")
or less restrictive and directly addressing the spdlog warning explicitly making the problem
xml_generator_config.append_cflags("-Wno-error=invalid-constexpr")

Reply:
just came to my mind, that I could try it be editing the file online.
Let's see what the CI makes out of it.
I LOVE multiple compilers on multiple systems :-)